activities.json

```diff
@@ -1231,6 +1231,19 @@
     "title": "WebUSB API",
     "url": "https://wicg.github.io/webusb/"
   },
+  {
+    "ciuName": "webxr",
+    "description": "This specification describes support for accessing virtual reality (VR) and augmented reality (AR) devices, including sensors and head-mounted displays, on the Web.",
+    "id": "webxr",
+    "mdnUrl": "https://developer.mozilla.org/en-US/docs/Web/API/WebXR_Device_API",
+    "mozBugUrl": "https://bugzilla.mozilla.org/show_bug.cgi?id=1419190",
+    "mozPosition": "worth prototyping",
+    "mozPositionDetail": "The WebXR Device API is the basis for VR, MR, and AR on the web.  This is a significant and large suite of features.  There is the potential for XR to provide significant benefits, but also a number of risks to individual privacy and security due to the way that XR relies on sensors and environmental data.  Developing new interaction models also presents challenges and considerable work is required before new norms are established.  Mozilla is actively working on WebXR and supportive of its overall goals.  Our participation in and implementation of this API is critical to understanding the feature and learning how to empower users in managing the associated risks.",
```

Comment:
can/should we mention that this is already shipped on Firefox Reality here?

Reply:
I like this positioning very much, thanks!

Reply:
@Manishearth, Kip's original PR mentioned that, but I think that it is better that we talk about what we've done without reference to a particular implementation.  Firefox Reality and Servo are both great ways for us to learn, but their singular focus (XR or research) means that I don't think we can claim that they are general purpose.  That's why I just kept this to "we're prototyping this and learning by doing that".

Reply:
That's fair. I think it's still a useful signal, though, if we mention it as being released in XR-specific browsers